articles/data-factory/copy-activity-log.md

```diff
@@ -99,6 +99,8 @@ After the copy activity runs completely, you can see the path of log files from
         }
 
 ```
+> [!NOTE]
+> It is not possible to provide a User defined name to the log files which are generated by enabling the `enableCopyActivityLog` property. `
```

Comment:
Please use lowercase on user-defined and remove the extra tick at the end of the line. 

```suggestion
> It isn't possible to provide a user-defined name for the log files which are generated by enabling the `enableCopyActivityLog` property.
```